dashboard/scripts/update_tts_i18n.rb

```diff
@@ -20,49 +18,57 @@ def text_translated?(left, right)
   clean(left) != clean(right)
 end
 
-TextToSpeech::VOICES.keys.each do |lang|
-  next if lang == :'en-US'
+def update_level_tts_i18n(level, script=nil)
+  # Short Instructions
 
-  I18n.locale = lang
-  puts "updating text-to-speech for #{I18n.locale}"
-  k1_scripts.each do |script|
-    script.levels.each do |level|
-      next unless level.is_a?(Blockly)
+  translated_text = level.tts_short_instructions_text
+  english_text = TextToSpeech.sanitize(level.short_instructions || "")
 
-      # Short Instructions
+  if translated_text.present? && text_translated?(translated_text, english_text)
+    level.tts_upload_to_s3(translated_text)
+  end
 
-      translated_text = level.tts_short_instructions_text
-      english_text = TextToSpeech.sanitize(level.short_instructions || "")
+  # Long Instructions
 
-      if text_translated?(translated_text, english_text)
-        level.tts_upload_to_s3(translated_text)
-      end
+  unless script && (script.csf_international? || script.twenty_hour?)
```

Comment:
It might be useful to have comments here explaining these exceptions

Reply:
it absolutely would be! In particular, if I had added comments here originally, I might remember now why we have these exceptions in the first place :upside_down_face: